charts/jenkins/values-casc-jobs.yaml.gotmpl

```diff
@@ -100,6 +100,10 @@ controller:
             folder('opensearch2')
         - script: > {{ tpl (readFile "jobs/opensearch2/nuxeo-opensearch2.groovy.gotmpl") . | nindent 12 }}
         - script: > {{ tpl (readFile "jobs/opensearch2/release-nuxeo-opensearch2.groovy.gotmpl") . | nindent 12 }}
+        - script: >
+            folder('elasticsearch9')
+        - script: > {{ tpl (readFile "jobs/elasticsearch9/nuxeo-elasticsearch9.groovy.gotmpl") . | nindent 12 }}
+        - script: > {{ tpl (readFile "jobs/elasticsearch9/release-nuxeo-elasticsearch9.groovy.gotmpl") . | nindent 12 }}
```

Comment:
Can you put them just before [this](https://github.com/nuxeo/platform-ci/blob/88381640b736fe923c01a6f301feff6f69f1b227/charts/jenkins/values-casc-jobs.yaml.gotmpl#L35) line, just for alpha sorting.